src/test/java/duke/task/TodoTest.java

```diff
@@ -0,0 +1,20 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TodoTest {
+
+    @Test
+    public void testStringConversion() {
```

Comment:
Very simple JUnit tests indeed (as mentioned in your commit message), but I feel that these test methods are essential to the overall correctness of your program. Very descriptive name used for test methods, making it very readable, great job!